recipes/rtm/all/test_package/test_package.cpp

```diff
@@ -1,3 +1,6 @@
+// workaround for a missing include in rtm
+#include <limits>
```

Comment:
Is this a bug in the code itself? Has it been fixed in any newer version/reported upstream?

Reply:
It's a bug in the project itself, yes. I have not created a bug repirt or PR yet, though.